framework/src/controller/application.js

```diff
@@ -288,13 +288,13 @@ class Application {
 	_compileAndValidateConfigurations() {
 		const modules = this.getModules();
 
-		this.config.app.nethash = this.genesisBlock.payloadHash;
```

Comment:
we can keep this as it was. For the framework, networkId is not the payloadHash.
We only need to change it in the `lisk-p2p`, for the framework, it can remain as it was for now.